### ISSUE TITLE
About Validation scheme

### ISSUE THREAD

**Issue by sybapp** [CVMI-Lab/SimGCD#5]
I would like to thank you for your contribution. However, I have some concerns regarding the validation strategy mentioned in your paper and implemented in the code. In the paper, you mentioned that the validation strategy involves using the old class samples from the validation set as shown in the following image:
![image](https://user-images.githubusercontent.com/28376371/230855174-7bf3a32f-2a6d-4492-97cc-d09cc157b6c4.png)
However, in the code, an unknown class selection model is used. I am wondering if this is an error or intentional. Here is the link to the relevant code:
https://github.com/CVMI-Lab/SimGCD/blob/47e8374f02534a45b423f8b52e364571a551cb27/train.py#L133

**Comment by xwen99** [comment]
Hi @sybapp, thanks for pointing out this issue! I have checked our development codebase, and verified that this was a mistake during the code refactoring. Please follow the description in the paper (and the original GCD codebase), and we will fix this issue soon. Thanks again!

**Comment by sybapp** [comment]
Is the result in the paper based on the performance of the last epoch?
